lib/Stats.js

```diff
@@ -1152,6 +1362,7 @@ class Stats {
 	}
 
 	static presetToOptions(name) {
+		// TODO :- DELETE THIS IN UPCOMING VERSION, WE've GOT PLUGINS :)
```

Comment:
Bad rebase?

Reply:
:smile: No, actulally, it's a static method, and it's used in some test cases individually,  https://github.com/webpack/webpack/blob/master/test/Stats.unittest.js#L162 . Couldn't remove it without breaking change. However we no longer need this except if some user specifically want to convert presetToOptions ( which i don't think should happen), so added to TODO to remove it in future version.

Reply:
Now this can be deleted, if suggested.. No function is using it internally, only test-cases are using. Suggestions? @sokra ??